integration/src/test/scala/ammonite/integration/StandaloneTests.scala

```diff
@@ -124,6 +124,83 @@ object StandaloneTests extends TestSuite{
       assert(e.toString.contains(expectedErrorMsg))
     }
 
+    'compilationErrorLineNumberTest1{
+      //Make sure correct line numbers are printed when an erroneous script is executed
+      val name = "compilationErrorLineNumberTest1.scala"
```

Comment:
Can you extract all the common code for these 4 test cases out into a helper function?